src/components/DAPAnalytics.jsx

```diff
@@ -0,0 +1,54 @@
+/* Component for handling DAP "virtual pageview" tracking mechanism */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Route } from 'react-router-dom';
+
+const scriptExists = url => document.querySelectorAll(`script[src="${url}"]`).length > 0;
+
+const logPageChange = (pathname, search = '') => {
+  const page = pathname + search;
+  const { location } = window;
+  // check if DAP script has been loaded
+  if (scriptExists('https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=NIH&subagency=NIAID')) {
+    // apply virtual pageview using DAP function
+    window.gas('send', 'pageview', `${location.origin}${page}`);
```

Comment:
in the documentation it doesn't look like they are adding the origin?

<img width="844" alt="Screen Shot 2020-12-01 at 2 05 44 PM" src="https://user-images.githubusercontent.com/4224001/100791061-5662bd80-33de-11eb-9a55-3376470aee60.png">

Reply:
cool, their manual didn't say it has to be without the origin but we can stick to their instructions

Reply:
i really don't know which one's right, just bringing it to your attention 😄

Reply:
yeah you are right actually, at least for GA the path should not have origin, so I think that also holds for DAP